c_cxx/Snpe_EP/README.md

```diff
@@ -0,0 +1,124 @@
+# Image classification with Inception v3 in CPP using SNPE Execution Provider
+1.  This image classification sample uses the Inception v3 model from [SNPE turtorial](https://developer.qualcomm.com/sites/default/files/docs/snpe/tutorial_inceptionv3.html) with DLC format. Wrap it as a custom node in an ONNX model to inference with SNPE Execution Provider.
+
+2.  The sample uses the Onnxruntime SNPE Execution Provider to run inference on various Qualcomm devices like Qualcomm CPU, GPU DSP, AIP, etc. It supports Windows ARM64, and Android.
+
+# Prerequisites (Linux host)
+1. Get a Linux host
+2. Download SNPE SDK from Qualcomm's developer site [here](https://developer.qualcomm.com/software/qualcomm-neural-processing-sdk)
+
+3. Setup SNPE on the Linux host. Setup environment for the tutorial. Follow the Tutorials and Examples for [Tensorflow inceptionv3](https://developer.qualcomm.com/sites/default/files/docs/snpe/tutorial_inceptionv3.html)
+4. Get the model [Inception v3](https://developer.qualcomm.com/sites/default/files/docs/snpe/tutorial_setup.html#tutorial_setup_inception_v3)
+
+    ```
+    python $SNPE_ROOT/models/inception_v3/scripts/setup_inceptionv3.py -a ~/tmpdir -d -r dsp
+    ```
+
+    The generated DLC file inception_v3.dlc and inception_v3_quantized.dlc can be found at $SNPE_ROOT/models/inception_v3/dlc/.
+
+    The data chairs.raw, notice_sign.raw, plastic_cup.raw and trash_bin.raw can be found at $SNPE_ROOT/models/inception_v3/data/. The sample applicatioin use these raw file as input.
+
+5. Create ONNX model from DLC file
+
+
+    Create a script gen_onnx_model.py in folder $SNPE_ROOT/models/inception_v3 with the code below:

```

Comment:
it would be great to create a generic version of this simple script and check into main repo. keep it simple and have it accept only a few params such as input/output name/type/shape and input/output model file names

Reply:
Is it worth to do that much? To make it generic we need to think about more, like support multi-input, multi-output. so, need to have a list for input names and shapes as parameter which may end up not easy to use, and make the code complicated. I think this script is simple enough and straight forward for user to make modifications for their model.